.github/workflows/node.js.yml

```diff
@@ -11,26 +11,4 @@ on:
 
 jobs:
   build:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        node-version: [18.x, 20.x]
-        os: [ubuntu-latest, windows-latest]
-
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-    - run: npm i --package-lock --package-lock-only
-    - run: npm ci
-    - run: npm run build --if-present
-    - run: npm test
-    - name: upload coverage
-      if: success()
-      uses: codecov/codecov-action@v3.1.1
-      with:
-        name: ${{ runner.os }} node.js ${{ matrix.node-version }}
+    uses: adobe/aio-reusable-workflows/.github/workflows/daily.yml@main
```

Comment:
this is wrong: should be uses: adobe/aio-reusable-workflows/.github/workflows/node.js.yml@main 
that's why the windows builds don't show up